resources/i18n/qqq.json

```diff
@@ -50,5 +50,129 @@
+	"find-in-page":"Represents the action of searching for a text in the current article/page.",
+	"set-fullscreen":"Represents the action of setting this desktop application to fill the entire computer screen.",
+	"quit-fullscreen":"Represents the action of setting this desktop application back to its original, non-fullscreen dimensions.",
+	"table-of-content":"Describes the table of content of an article.",
+	"reading-list":"Describes a list of articles save for future reading.",
+	"reading-list-title":"Describes a list of articles save for future reading.",
+	"zoom-in":"Represents the action of zooming into the article, making the article content appear larger.",
+	"zoom-out":"Represents the action of zooming out of the article, making the article content appear smaller.",
+	"zoom-reset":"Represents the action of resetting the article zoom, changing the article content size back to default",
+	"feedback":"{{identical|Feedback}}",
+	"report-a-bug":"Represents the action of reporting a bug in this desktop application.",
+	"request-a-feature":"Represents the action of requesting a new feature in this desktop application.",
+	"about-kiwix":"Describes the about page of Kiwix, containing information of the organization.",
+	"donate-to-support-kiwix":"Represents the action of donating to support the Kiwix Organization.",
+	"exit":"Represents the action of exiting the desktop application",
+	"save-file-as-window-title":"Represents the action of saving a file with its name based on the window title or a chosen name.",
+	"download-finished-message":"Text displayed when the download action has been completed",
+	"tools":"{{Identical|Tools}}",
+	"window-title":"The title text displayed when the library browser is present",
+	"browse-directory":"Represents the action of browsing the local file system for a directory.",
+	"settings":"{{Identical|Settings}}",
+	"forward":"{{Identical|Forward}}",
+	"search-files":"Represents the action of searching online or locally for Zim files.",
+	"content-type":"Describes the type of contents available on a Zim file, for example, text or pictures.",
+	"reset-sort":"Represents the action of resetting the sorting of the items to default order.",
+	"open":"{{Identical|Open}}",
+	"download":"{{Identical|Download}}",
+	"resume":"\"Resume\" here refers to continuing a task.",
+	"port-for-local-kiwix-server-setting":"Describes the port of the computer of which the local Kiwix server will be hosted on.",
+	"zoom-level-setting":"Describes the default zoom level of articles displayed on this desktop application.",
+	"download-directory-setting":"Describes the directory in the file system where Zim files will be downloaded to.",
+	"monitor-directory-setting":"Describes the directory in the file system where we will monitor for Zim file changes.",
+	"monitor-directory-default":"Describes the default directory in the file system where we will monitor for Zim file changes.",
```

Comment:
I think that this is no longer used. We should rather drop such entries from `en.json`